Doc/library/pickle.rst

```diff
@@ -788,14 +786,16 @@ the code ::
    f = io.BytesIO()
    p = MyPickler(f)
 
-does the same, but all instances of ``MyPickler`` will by default
-share the same dispatch table.  The equivalent code using the
-:mod:`copyreg` module is ::
+does the same but all instances of ``MyPickler`` will by default
+share the private dispatch table.  On the other hand, the code ::
 
    copyreg.pickle(SomeClass, reduce_SomeClass)
    f = io.BytesIO()
    p = pickle.Pickler(f)
 
+does the same but all instances of :class:`pickle.Pickler` will by default
+share the modified global dispatch table.
```

Comment:
Is this change intentional? Doesn't look like a typo fix, though I agree that the old wording doesn't make much sense. However, I would word it as 
```suggestion
modifies the global dispatch table shared by all users of the :mod:`copyreg` module.
```

Reply:
Yes this was intentional but your wording is better, thanks!